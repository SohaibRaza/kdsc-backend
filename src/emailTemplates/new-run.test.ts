import newRun from "./new-run"

describe("email-templates/new-run", () => {
  test("should return new Run Email Subject", () => {
    expect(newRun.subject({ checklistName: "Test Run" })).toBe("Due: Test Run")
  })

  test("should return new Run Email Content", () => {
    expect(
      newRun.content({ checklistName: "Test Run", runUrl: "www.google.com" })
    ).toMatchSnapshot()
  })
})
