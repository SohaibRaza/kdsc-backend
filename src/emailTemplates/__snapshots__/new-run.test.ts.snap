// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`email-templates/new-run should return new Run Email Content 1`] = `
"<td class=\\"wrapper\\">
    <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
        <tr>
            <td>
                <p>Hello there!</p>
                <p>The checklist <strong>Test Run</strong> has been created. You can complete this checklist via the following URL:</p><br/>
                <table class=\\"btn btn-primary\\" role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                    <tbody>
                        <tr>
                            <td align=\\"left\\">
                                <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                                    <tbody>
                                        <tr>
                                            <td><a href=\\"www.google.com\\" target=\\"_blank\\">Open Run</a></td>
                                        </tr>
                                    </tbody>
                                </table>
                            </td>
                        </tr>
                    </tbody>
                </table><br/>
                <p>Thanks!</p>
                <p>The KDSP team</p>
            </td>
        </tr>
    </table>
    </td>"
`;
